packages/flutter_tools/lib/src/http_host_validator.dart

```diff
@@ -0,0 +1,126 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:webdriver/async_core.dart';
```

Comment:
What is this for? I think you need to `import 'dart:async'` for `TimeoutException`.